src/helpers/message_helpers.py

```diff
@@ -0,0 +1,32 @@
+from typing import List
+
+from src.helpers.constants import BOX_WRAPPER_CHAR
+from src.helpers.constants import CODE_BLOCK
+from src.helpers.constants import EXPERIENCE_COLUMN_NAME
+from src.helpers.constants import MINIMUM_SPACING
+from src.helpers.constants import POST_TEXT
+from src.helpers.constants import PRE_TEXT
+from src.helpers.constants import QUEST_COLUMN_NAME
+from src.helpers.constants import WRAPPER_TEXT_LEN
+from src.models import Quest
+
+
+def _create_single_quest_line(first_column: str, second_column: str, line_length: int) -> str:
+    num_spaces_needed = line_length - WRAPPER_TEXT_LEN - len(first_column) - len(second_column)
+    single_line = "".join([PRE_TEXT, first_column, " " * num_spaces_needed, second_column, POST_TEXT])
+    return single_line
+
+
+def format_quest_board(quests: List[Quest]) -> str:
```

Comment:
i didnt make this async for ease of testing, but should this get asynced? Or maybe should consider paginating the quest lists

Reply:
This could also use a doc string

Reply:
Added a doc string, will address needing to paginate in the future when needed